qiskit_ibm_provider/job/ibm_circuit_job.py

```diff
@@ -657,11 +571,7 @@ def backend_options(self) -> Dict[str, Any]:
             Backend options used for this job. An empty dictionary
             is returned if the options cannot be retrieved.
         """
-        qobj = self._get_qobj()
-        if not qobj:
-            return {}
-        _, options, _ = disassemble(qobj)
-        return options
+        return self._params or {}
```

Comment:
This should filter out `header` and `circuits` since they are not part of options.